README.md

```diff
@@ -72,7 +86,7 @@ When using `DOCKER_CERT_PATH_HOST` you must also add the following under `volume
 ```
 {"9614045":"syn16799953"}
 ```
-- `TOIL_CLI_OPTIONS` - (optional) Space separated list of options.  See https://toil.readthedocs.io/en/3.15.0/running/cliOptions.html.  Example:
+- `TOIL_CLI_OPTIONS` - (optional, but highly recommended) Space separated list of options. (Without the toil parameters, you may run into errors with running the workflow hook).  See https://toil.readthedocs.io/en/3.15.0/running/cliOptions.html.  Example:
```

Comment:
"with running the workflow hook" 
change to
"when a new workflow job is started"